src/content/docs/cypher/data-definition/create-table.md

```diff
@@ -75,23 +75,48 @@ Once you create node tables, you can define relationships between them using the
+- **Syntax**: There is no comma between the `FROM` and `TO` clauses, however a comma is needed between two node table pairs.
 - **Directionality**: Each relationship has a direction following the property graph model. So when `Follows` relationship records are added, each one has a specific source (FROM) node and a specific destination (TO) node.
 - **Primary keys**: You cannot define a primary key for relationship records. Each relationship gets a unique system-level edge ID, which are internally generated. You can check if two edges are the same, i.e., have the same edge ID, using the `=` and `!=` operator between the `ID()` function on two variables that bind to relationships. For example, you can query `MATCH (n1:User)-[r1:Follows]->(n2:User)<-[r2:Follows]-(n3:User) WHERE ID(r1) != ID(r2) RETURN *` to ensure that the same relationship does not bind to both r1 and r2.
-- **Pairing**: A relationship can only be defined as being from one node table/label to one node table/label.
-  :::
+:::
 
-### Relationship Multiplicities
+### Bulk insert to relationship table with multiple from-to pairs
 
-For any relationship label E, e.g., , by default there can be multiple relationships from any node v both in the forward and backward direction. In database terminology, relationships are by default many-to-many. For example in the first Follows example above: (i) any User node v can follow multiple User nodes; and (ii) be followed by multiple User nodes. You can also constrain the multiplicity to _at most 1_ (we don't yet support exactly 1 semantics as in foreign key constraints in relational systems) in either direction. You can restrict the multiplicities for two reasons:
+Internally, a relationship table with multiple from-to pairs will create a child table per from-to pair. In the example above, the following two children
+tables will be created internally
+```
+Knows_User_User
+Knows_User_City
+```
 
-1. Constraint: Multiplicities can serve as constraints you would like to enforce (e..g, you want Kùzu to error if an application tries to add a second relationship of a particular label to some node)
-2. Performance: Kùzu can store 1-to-1, many-to-1, or 1-to-many relationships (explained momentarily) in more efficient/compressed format, which is also faster to scan.
+When bulk insert into a relationship table with multiple from-to pairs, user need to specify which child table to insert through `from, to` options. For exmaple, the following two statements will bulk insert into `Knows` relationship table.
+```
+Copy Knows FROM 'knows_user_user.csv' (from='User', to='User');
+Copy Knows FROM 'knows_user_city.csv' (from='User', to='City');
+```
+
+Alternatively, you can also directly copy into a child table. Though this approach is not recommended and will be deprecated in the future release.
+```
+Copy Knows_User_User FROM 'knows_user_user.csv';
+Copy Knows_User_City FROM 'knows_user_city.csv';
+```
+
+### Relationship Multiplicities
```

Comment:
We should also mention that the multiplicity constraint applies to all pairs of FROM and TO defined in a rel table now.

Reply:
@acquamarin do you know of a good way to describe this? Please post here and I can figure out where to put it.

Reply:
i think a good way is to add after this doc:
```
You can optionally declare the multiplicity of relationships by adding `MANY_MANY`, `ONE_MANY`, `MANY_ONE`, or `ONE_ONE` clauses to the end of the `CREATE REL TABLE` command.
```
The multiplicity definition applies to all from-to pairs in the rel table.